internal/storage/localdir.go

```diff
@@ -17,9 +17,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/operator-framework/rukpak/internal/util"
+	"github.com/operator-framework/rukpak/pkg/storage"
```

Comment:
We may want/need to be consistent with the package import naming here.

Reply:
OK.  I put the same name as the others. `pkgstorage`